sessionctx/variable/session.go

```diff
@@ -910,6 +914,19 @@ func (s *SessionVars) SlowLogFormat(txnTS uint64, costTime time.Duration, execDe
 	if len(digest) > 0 {
 		buf.WriteString(SlowLogRowPrefixStr + SlowLogDigestStr + SlowLogSpaceMarkStr + digest + "\n")
 	}
+	if len(statsInfos) > 0 {
+		buf.WriteString(SlowLogRowPrefixStr + SlowLogStatsInfoStr + SlowLogSpaceMarkStr)
+		firstComma := false
+		for k, v := range statsInfos {
+			if firstComma {
+				buf.WriteString("," + k + ":" + strconv.FormatUint(v, 10))
```

Comment:
should we write `pseudo` instead of 0 to be consistent with `explain`?

Reply:
done.